x-pack/platform/plugins/shared/stack_alerts/server/rule_types/es_query/executor.ts

```diff
@@ -193,6 +189,7 @@ export async function executor(core: CoreSetup, options: ExecutorOptions<EsQuery
         [ALERT_EVALUATION_THRESHOLD]: params.threshold?.length === 1 ? params.threshold[0] : null,
         ...ecsGroups,
```

Comment:
@cesco-f Can you check @benakansara 's comment [here](https://github.com/elastic/kibana/pull/228754/files#r2222988625)? Based on her comment, we should use `groupings` object in `getEcsGroups(groups)` function. Since `getEcsGroup` function is shareable by multiple rules, we might need to separate it out for the rule we are refactoring and once all rules are done we can make it shareable again.

@benakansara Same applies to all rule types, right?